nss-day-cohort-45/nutyshell-leaving-the-lights-on-1
# Events Section & CSS

# Description
This is the events section, it meets MVP, aside from Show Weather.
This is all of the CSS, which included updates to index.html, main.css, and both auth components

Fixes # (issue)
## Type of change


# Testing Instructions
All pages should include header w/ default img
When logged in, login box & register box should disappear
When logged in, Tasks, News and Events should populate

# Checklist:
